crates/viewer/re_renderer/src/config.rs

```diff
@@ -61,7 +61,17 @@ impl DeviceTier {
             flags: match self {
                 Self::Gles => wgpu::DownlevelFlags::empty(),
                 // Require fully WebGPU compliance for the native tier.
-                Self::FullWebGpuSupport => wgpu::DownlevelFlags::all(),
+                Self::FullWebGpuSupport => {
+                    // Turn a blind eye on a few features that are missing as of writing in WSL even with latest Vulkan drivers
+                    // and pretend we still have full WebGPU support anyways.
+                    wgpu::DownlevelFlags::compliant()
+                        // Lack means that we can't set the format of views on surface textures (the result of `get_current_texture`) surface won't tell us which formats are supported.
+                        // We avoid doing anything wonky with surfaces anyways, so we won't hit this.
+                        .intersection(wgpu::DownlevelFlags::SURFACE_VIEW_FORMATS.complement())
+                        // Lack means we only get 2^24-1 indices for drawing.
+                        // Typically we don't reach this limit.
```

Comment:
What happens if we do?

Reply:
hum. excellent question. I'll find out!

Reply:
seems to be undefined behavior, couldn't find anything on it.
Clairifed comment though a bit: This is **not** the number of indices but the value for the indices, i.e. the vertex count is limited to 16.8mio if this flag is not present on the adapter